File: v_1.0.2/source/hin-common/src/main/java/com/hin/domain/vo/ProductVO.java
/**
 * 
 */
package com.hin.domain.vo;

/**
 * @author krishna.lr
 * 
 */
public class ProductVO {
	private String messageId;
	private String messageTitle;
	private String messageStatus;
	private String createdDate;
	private String isActive;
	private String message;
	/**
	 * @return the messageId
	 */
	public String getMessageId() {
		return messageId;
	}
	/**
	 * @param messageId the messageId to set
	 */
	public void setMessageId(String messageId) {
		this.messageId = messageId;
	}
	/**
	 * @return the messageTitle
	 */
	public String getMessageTitle() {
		return messageTitle;
	}
	/**
	 * @param messageTitle the messageTitle to set
	 */
	public void setMessageTitle(String messageTitle) {
		this.messageTitle = messageTitle;
	}
	/**
	 * @return the messageStatus
	 */
	public String getMessageStatus() {
		return messageStatus;
	}
	/**
	 * @param messageStatus the messageStatus to set
	 */
	public void setMessageStatus(String messageStatus) {
		this.messageStatus = messageStatus;
	}
	/**
	 * @return the createdDate
	 */
	public String getCreatedDate() {
		return createdDate;
	}
	/**
	 * @param createdDate the createdDate to set
	 */
	public void setCreatedDate(String createdDate) {
		this.createdDate = createdDate;
	}
	/**
	 * @return the isActive
	 */
	public String getIsActive() {
		return isActive;
	}
	/**
	 * @param isActive the isActive to set
	 */
	public void setIsActive(String isActive) {
		this.isActive = isActive;
	}
	/**
	 * @return the message
	 */
	public String getMessage() {
		return message;
	}
	/**
	 * @param message the message to set
	 */
	public void setMessage(String message) {
		this.message = message;
	}
	/* (non-Javadoc)
	 * @see java.lang.Object#toString()
	 */
	@Override
	public String toString() {
		return "ProductVO [messageId=" + messageId + ", messageTitle="
				+ messageTitle + ", messageStatus=" + messageStatus
				+ ", createdDate=" + createdDate + ", isActive=" + isActive
				+ ", message=" + message + "]";
	}
}

File: v_1.0.2/source/hin-web/src/main/webapp/html/js/vo/ProductVO.js
var HIN;
if (!HIN)
	HIN = {};

HIN.ProductVO = function() {
	this.messageId = null;
	this.messageTitle = null;
	this.messageStatus = null;
	this.createdDate = null;
	this.isActive = null;
	this.message = null;
	this.doseQuantity = null;
	this.unitQuantity = null;
};

HIN.ProductVO.prototype.setMessage = function(message) {
	this.message = message;
	var value = XmlUtil.getXPathResult(this.message,
			AppConstants.XPaths.Product.DOSAGE_QUANTITY ,
			XPathResult.STRING_TYPE);
	this.doseQuantity = (value && value.stringValue) ? value.stringValue: "";
	
	var unit = XmlUtil.getXPathResult(this.message,
			AppConstants.XPaths.Product.DOSAGE_UNIT_QUANTITY ,
			XPathResult.STRING_TYPE);
	this.unitQuantity = (unit && unit.stringValue) ? unit.stringValue: "";
}
;

HIN.ProductVO.prototype.toString = function() {
	return "[" + this.messageId + " , "+ this.messageTitle + " , "+ this.messageStatus + " , "+ this.createdDate + " , " + this.doseQuantity + " , "+ this.isActive + " , "+ this.unitQuantity + "]";
};
